### ISSUE TITLE
Recived Unknown synced data key

### ISSUE THREAD

**Issue by alezhshede** [MrCrayfish/Obfuscate#29]
im just trying to play around with your mods but when i join my server it says 

![image](https://user-images.githubusercontent.com/82260461/114263611-e21cd400-99e6-11eb-9188-a495d8ca1d97.png)

is there anyway to fix this?

-alexander

**Comment by idkanynames** [comment]
I wish I knew ;-;

**Comment by lvavila** [comment]
Yeah I get the same error.  I tried downgrading but its a domino effect as timeless_and_classics-gun_mod relies on the newest Obfuscate.  So I go without MyCrayfish :(
